src/dbt_client/index.ts

```diff
@@ -25,11 +27,26 @@ export class DBTClient implements Disposable {
     this._onDBTInstallationVerificationEvent,
   ];
   private shownError = false;
+  private dbtDetection: DBTDetection;
 
   constructor(
     private pythonEnvironment: PythonEnvironment,
     private dbtCoreDetection: DBTCoreDetection,
-  ) {}
+    private dbtCloudDetection: DBTCloudDetection,
+  ) {
+    const dbtIntegrationMode = workspace
```

Comment:
should we think about multi project setting as well? If we have to support multi project, then this should go under resource level config